### PR TITLE
Adding automated to edxapp and discovery plays

### PR DESCRIPTION
**Pull request by e0d**
## Configuration Pull Request

Make sure that the following steps are done before merging
- [ ] @devops team member has commented with :+1:
- [ ] are you adding any new default values that need to be overriden when this goes live?  
  - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
  - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by e0d** [comment]
@jibsheet

**Comment by jibsheet** [comment]
👍 but would be good to tack a little doc into defaults/main.yml of each role

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -351,20 +351,18 @@ EDXAPP_PYTHON_SANDBOX: true
...
-- "ALL=({{ common_web_user }})  NOPASSWD:SETENV:{{ COMMON_BIN_DIR }}/python.edxapp {{ COMMON_BIN_DIR }}/manage.edxapp lms transfer_students *"
+# Vars that are used when the automated role is "mixed-in" via the deploying play
+# This data structure specifies all the users with access to run command remotely
+# over SSH and the fully qualified command that they can run via sudo to the
+# application user
+
+EDXAPP_AUTOMATED_USERS:
+  automated_user:
+    sudo_commands:
+      - command: "{{ edxapp_venv_bin }}/python {{ edxapp_code_dir }}/manage.py lms migrate --list --settings=aws"
```
Comment (jibsheet) [inline playbooks/roles/edxapp/defaults/main.yml:362]: Are we allowing migration?  For testing?  I thought the real eventual command was the program refreshing?

**Comment by jibsheet** [comment]
@e0d Just a question about what commands you're allowing, otherwise 👍

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (e0d) [inline playbooks/roles/edxapp/defaults/main.yml:362]: One can allow arbitrary commands, I adopted the example from the removed, prior version of the config.  We are not currently planning to use this for migrations.

**Pull request merged**